pkg/Build/index.html

```diff
@@ -3,6 +3,18 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no, viewport-fit=cover">
 <meta name="apple-mobile-web-app-capable" content="yes">
+
+<head>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-KXNNFC60KV"></script>
```

Comment:
I find this a little off-putting wrt our user-sovereignty story. 

I'm not sure how to square this circle, but we should discuss.

Reply:
discussed :)